src/main/webapp/WEB-INF/tags/shared/commentsPanel.tag

```diff
@@ -21,65 +21,89 @@
 </c:choose>
 <div class="panel panel-primary">
     <div class="panel-heading">
+        <div class='display-icon pull-right'>
+            <span class="glyphicon ${'glyphicon-chevron-up'} pull-right"></span>
+        </div>
         <strong>${bodyTitle}</strong>
     </div>
-    <div class="panel-body">
+    <div class="panel-collapse collapse in">
         <c:if test="${viewingDraft}">Your comments that are not finished:</c:if>
-        <c:set var="commentIndex" value="${0}"/>
-        <c:forEach items="${commentsForStudentsTables}" var="commentsForStudentsTable">
-            <div class="panel panel-info student-record-comments${commentsForStudentsTable.extraClass}"
-                 <c:if test="${empty commentsForStudentsTable.rows && not forRecordsPage}">style="display: none;"</c:if>>
+        <c:set var="commentIndex" value="${0}" />
```

Comment:
The rest of the changes below in this file all seem to be formatting changes. do revert them. If you find that other parts of the code requires reformatting or refactoring, you could create an issue for it and work on it in a separate PR.